src/LodeRunner/src/config.cs

```diff
@@ -214,6 +223,23 @@ public LogLevel LogLevel
         /// </value>
         public string UrlPrefix { get; set; }
 
+        /// <summary>
+        /// Gets the client identifier and test run identifier message.
+        /// </summary>
+        /// <returns>
+        /// Concatenated string describing ClientId and TestRunId information.
+        /// </returns>
+        public string GetClientIdAndTestRunIdInfo()
+        {
+            string testRunId = string.Empty;
+            if (!string.IsNullOrEmpty(TestRunId))
+            {
+                testRunId = $", {SystemConstants.LogTestRunIdPrefix}{TestRunId})";
+            }
+
+            return $"[{SystemConstants.LogClientIdPrefix}{this.ClientStatusId}){testRunId}]";
```

Comment:
It looks like we're logging the client status ID and not the client ID (loadClientId). I think I remember an issue that included loadClientId in log analytics, which makes me think maybe this should be loadClientId. I believe @kforeverisback worked on it, in case I'm mis-remembering.

Reply:
Grafana and Azure monitoring are showing client status ID

Reply:
the log entry is now displaying ClientStatusId, LoadClientId and Test Run